### PR TITLE
Rework itc list gathering

### LINKED ISSUE #309: Rework ITC device list gathering and DAEPhys panel layout
- [x]  HW_ITC_ListDevices: Use direct close call to avoid `ITCGetState2` call which seems to crash ITC1600 hardware from time to time
- [x] combined controls in DAEphys panel for device type and number. Only show the available ones.
- [x] fetch HW_ITC_ListDevices once, save across test cases
- [x] Remove "open device" + "device_number"
- [x] Make one ITC device from every type available in evil mode

### PR DESCRIPTION
**Pull request by t-b**
Close #309.

### CODE REVIEW

**Comment by timjarsky** [comment]
@t-b 

- [ ] make config work with reworked list gathering

- [ ] add a small refresh button so user can initialize a hardware scan
image


- [ ] reduce size of group box control `group_Hardware_FolderPath`

**Comment by t-b** [comment]
@timjarsky Config file works here, what is your error?

**Comment by timjarsky** [comment]
> @timjarsky Config file works here, what is the error you are seeing?
it looks like the device is already open
image

File: Packages/MIES/MIES_DAEphys_Macro.ipf [hunk 1]
```diff
@@ -3616,6 +3602,9 @@ Window DA_Ephys() : Panel
 	GroupBox group_acq_tp,userdata(tabnum)=  "5",userdata(tabcontrol)=  "ADC"
 	GroupBox group_acq_daq,pos={31.00,554.00},size={208.00,62.00},disable=1,title="Data Acquisition"
 	GroupBox group_acq_daq,userdata(tabnum)=  "5",userdata(tabcontrol)=  "ADC"
+	Button button_hardware_rescan,pos={40.00,75.00},size={37.00,36.00},proc=ButtonProc_Hardware_rescan,title="\\Z20🔄"
```
Comment (timjarsky) [inline Packages/MIES/MIES_DAEphys_Macro.ipf]: @t-b try this for the title "\\Z24\\F'Wingdings 3'P"

**Comment by timjarsky** [comment]
@t-b 

- [ ] add refresh button to tab control

**Comment by t-b** [comment]
grafik

File: Packages/MIES/MIES_DAEphys_Macro.ipf (hunk 1 above)
Comment (t-b) [reply Packages/MIES/MIES_DAEphys_Macro.ipf]: I don't have that font. I'll make three proc pictures of mine. Also disable the button after locking.

Comment (t-b) [reply Packages/MIES/MIES_DAEphys_Macro.ipf]: Done.

**Comment by t-b** [comment]
@timjarsky Ready. Feel free merge as well if it is good.

**Pull request merged**